src/components/about/FAQCard.tsx

```diff
@@ -9,41 +8,37 @@ interface FAQCardProps {
 }
 
 const FAQCard: React.FC<FAQCardProps> = ({ text, backText }) => {
-  const [flipped, setFlipped] = useState(false);
-
   return (
     <motion.div
-      whileHover={{ scale: 1.1 }}
+      whileHover={{ scale: 1.1, rotateY: 180 }}
       transition={{ duration: 0.5 }}
-      animate={{ rotateY: flipped ? 180 : 0 }}
-      onClick={() => setFlipped((prevState) => !prevState)}
       className="relative m-4 flex cursor-pointer flex-col items-center"
       style={{ transformStyle: "preserve-3d" }}
     >
       <div
         className="top pb-5 text-4xl text-leap-mid-green md:text-5xl"
         style={{ position: "relative", transformStyle: "preserve-3d" }}
       >
-        <Heart style={{ backfaceVisibility: "hidden" }}></Heart>
+        <Heart style={{ backfaceVisibility: "hidden" }} />
         <Heart
           fill="#4F8C6C"
           className="absolute inset-0 flex items-center justify-center"
           style={{ backfaceVisibility: "hidden", transform: "rotateY(180deg)" }}
-        ></Heart>
+        />
       </div>
 
       <div
         className="square-lg flex aspect-[4/3] w-64 items-center justify-center bg-leap-mid-green p-6 text-center font-leap text-xl text-white shadow-lg shadow-black md:w-72 md:text-lg lg:w-80 lg:text-2xl"
         style={{ position: "relative", transformStyle: "preserve-3d" }}
       >
         <div
-          className="absolute flex h-full w-full items-center justify-center px-1"
+          className="absolute flex h-full w-full items-center justify-center px-1 leading-loose"
```

Comment:
```suggestion
          className="absolute flex h-full w-full items-center justify-center px-1"
```
leading doesn't fix the issue we have with the text